Tests/SentryTests/PrivateSentrySDKOnlyTests.swift

```diff
@@ -3,6 +3,21 @@ import XCTest
 
 class PrivateSentrySDKOnlyTests: XCTestCase {
 
+    private class Fixture {
```

Comment:
`m`: What made you choose to add these tests here instead of adding them to SentrySDKTests or SentryHubTests where the logic is?

Reply:
Because this is the interface hybrid SDKs use. 

The underlying structure can change as long as session is handled as expected.